### ISSUE TITLE
crash on expression

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#474]
The following crashes using v3.1.1

```xml
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mrow data-mjx-texclass="VCENTER">
    <mglyph width="171pt" height="11pt" src="Images/imgdb0e63face2e663d9305f03b8be3fce8.svg"></mglyph>
  </mrow>
  <mo>.</mo>
</math>
```

with xmldom errors:

```
RangeError: Maximum call stack size exceeded
    at Array.join (<anonymous>)
    at getTextContent (.../xmldom-sre/dom.js:1283:16)
... (more xmldom trace)
```

**Comment by pkra** [comment]
Ah, this seems to be fixed in 3.2.0-beta2. Sorry for the noise.

**Comment by pkra** [comment]
Here's another one that's fixed in 3.2.0-beta2 in case it's helpful:

```xml
<math xmlns="http://www.w3.org/1998/Math/MathML">
  <msqrt>
    <mrow>
      <mpadded width="0">
        <mphantom>
          <mo stretchy="false">(</mo>
        </mphantom>
      </mpadded>
    </mrow>
  </msqrt>
  <msub>
    <mi>I</mi>
    <mrow>
      <mi>&#x3BC;</mi>
    </mrow>
  </msub>
  <mo>=</mo>
  <mo stretchy="false">(</mo>
  <mi>x</mi>
  <mo>,</mo>
  <mi>y</mi>
  <mo stretchy="false">)</mo>
</math>
```
